src/Instance/InstanceDetails/InstanceAcquisition/useInstanceAcquisition.js

```diff
@@ -54,7 +54,8 @@ const useInstanceAcquisition = (id) => {
       });
       const ordersMap = hydratedOrders.reduce((acc, order) => ({ ...acc, [order.id]: order }), {});
 
-      return poLines.map(line => ({ ...line, order: ordersMap[line.purchaseOrderId] }));
+      return orderBy(poLines.map(line => ({ ...line, order: ordersMap[line.purchaseOrderId] })),
+        ({ order }) => order?.metadata?.createdDate, ['desc']);
```

Comment:
there is dateOrdered columns, why is createdDate used?

Reply:
in this case dateOrdered is the same as createdDate

Reply:
Thanks I'll update